### PR TITLE
Add information about the addition of a config.json

### PR DESCRIPTION
**Pull request by iHiD**
The PR is self-documenting. #22 shows a sample implementation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,29 +47,32 @@ For an equality assertion failure, the error with context might look as follows:
...
+If you would like to provide a custom message (rather than the default interpolated message that we provide), you can also add a "msg" key, which must contain the command, and should in most cases also embed the [test runner's][test-runner-output-format] output (usually the assertion error message) in the message by using `%{output}`. For example:
 
 ```json
-{
-  "MyTestMethod": {
-    "cmd": "MyTests.MyTestMethod(1)"
+tests: [
+  {
+    "name": "MyTestMethod",
+    "cmd": "MyTests.MyTestMethod(1)
```
Comment (iHiD) [inline README.md]:
```suggestion
    "cmd": "MyTests.MyTestMethod(1),
```

File: README.md [hunk 2]
```diff
@@ -36,29 +49,36 @@ For an equality assertion failure, the error with context might look as follows:
...
+      "cmd": "We tried running `MyTests.MyTestMethod(1)` but got the following error:\n\n%{output}"
+    }
+  ]
 }
 ```
 
-This definition is functionally equivalent to the explicit version that specified the full message.
+### Test order
+
+We only show one error message to a user when the tests fail. If multiple tests fail, we show the first test that is listed in the `tests` array of the `.meta/config.json`. It is therefore important to structure the `tests` array from easiest to hardest (simple cases first, edge cases last).
```
Comment (iHiD) [inline README.md]:
```suggestion
We only show one error message to a user when the tests fail. If multiple tests fail, we show the first test that is listed in the `tests` array of the `.meta/config.json`. It is therefore important to structure the `tests` array from easiest to hardest (simple cases first, edge cases last).

We also use this file to guarantee at a website level that the correct tests have run. By checking the results.json against this list of tests, we can confirm that the user hasn't bypassed the test-running.
```

File: README.md [hunk 3]
```diff
@@ -36,29 +49,38 @@ For an equality assertion failure, the error with context might look as follows:
...
 ```json
 {
-  "MyTestMethod": {
-    "cmd": "MyTests.MyTestMethod(1)"
-  }
+  "tests": [
+    {
+      "name": "MyTestMethod",
+      "cmd": "MyTests.MyTestMethod(1)",
+      "cmd": "We tried running `MyTests.MyTestMethod(1)` but got the following error:\n\n%{output}"
```
Comment (ErikSchierboom) [inline README.md]:
```suggestion
      "msg": "We tried running `MyTests.MyTestMethod(1)` but got the following error:\n\n%{output}"
```

We could consider also having a `%{cmd}` placeholder available, as "also add a "msg" key" seems to imply that the `cmd` key is required.

```suggestion
      "msg": "We tried running `%{cmd}` but got the following error:\n\n%{output}"
```

File: README.md [hunk 4]
```diff
@@ -36,29 +49,38 @@ For an equality assertion failure, the error with context might look as follows:
...
+  ]
 }
 ```
 
-This definition is functionally equivalent to the explicit version that specified the full message.
+### Test order
+
+We only show one error message to a user when the tests fail. If multiple tests fail, we show the first test that is listed in the `tests` array of the `.meta/config.json`. It is therefore important to structure the `tests` array from easiest to hardest (simple cases first, edge cases last).
+
+We also use this file to guarantee at a website level that the correct tests have run. By checking the results.json against this list of tests, we can confirm that the user hasn't bypassed the test-running.
```
Comment (ErikSchierboom) [inline README.md]:
```suggestion
We also use this file to guarantee at a website level that the correct tests have run. By checking the `results.json` against this list of tests, we can confirm that the user hasn't bypassed the test-running.
```

File: README.md [hunk 5]
```diff
@@ -36,29 +49,38 @@ For an equality assertion failure, the error with context might look as follows:
...
+      "cmd": "We tried running `MyTests.MyTestMethod(1)` but got the following error:\n\n%{output}"
+    }
+  ]
 }
 ```
 
-This definition is functionally equivalent to the explicit version that specified the full message.
+### Test order
+
+We only show one error message to a user when the tests fail. If multiple tests fail, we show the first test that is listed in the `tests` array of the `.meta/config.json`. It is therefore important to structure the `tests` array from easiest to hardest (simple cases first, edge cases last).
```
Comment (ErikSchierboom) [inline README.md]:
```suggestion
We only show one error message to a user when one or more tests fail. If multiple tests fail, we show the first test that is listed in the `tests` array of the `.meta/config.json`. It is therefore important to structure the `tests` array from easiest to hardest (simple cases first, edge cases last).
```
